src/applications/personalization/dashboard/containers/YourApplications.jsx

```diff
@@ -65,6 +67,27 @@ class YourApplications extends React.Component {
   }
 }
 
+YourApplications.propTypes = {
```

Comment:
I just moved this up so it wasn't sitting between `mapStateToProps` and `mapDispatchToProps`